ML-People/backend-express
# initialize express server

To test on local environment:

1) npm install 
2) npm start

terminal should give message about listening on port 3030
